aca_lts_eval.py

```diff
@@ -176,10 +177,45 @@ def select_ri_guide_stars(ra, dec, cone_stars):
     return G_RI_CAT_CACHE[id_key]
 
 
-def t_lose_star(mag, ref_mag=10.3, ref_tccd=-11.5, scale_4c=None):
+def guide_count(mags, tccd=-10.2):
```

Comment:
`t_ccd` should not have a default really.  There is nothing unique or special about -10.2 when calling this function.

Reply:
And call the kwarg `t_ccd`, same below.

Reply:
OK.  I thought for general purpose use, having this set at the planning limit made some sense, but that was just lazy in this implementation.

Reply:
This won't be the planning limit in 6 months.

Reply:
And I was already agreeing this was lazy in this routine, but also keep in mind that the code already has things I need to update manually when the planning limit changes.